src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -65,6 +65,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_format', help="Indicates whether the logs should be displayed in raw format", action='store_true')
         c.argument('os_type', options_list=['--os'], help='The operating system type required for the build.', arg_type=get_enum_type(OsType), deprecate_info=c.deprecate(redirect='platform', hide=True))
         c.argument('platform', help="The platform where build/task is run, Eg, 'windows' and 'linux'. When it's used in build commands, it also can be specified in 'os/arch/variant' format for the resulting image. Eg, linux/arm/v7. The 'arch' and 'variant' parts are optional.")
+        c.argument('target', help='The name of the target build stage.')
```

Comment:
Not sure if it should go at the root or just copy/paste to `acr build` and `acr task`, it seems like we sorta have conflicting places where stuff is already. I.e., `no_push` is similar to `target` and is copy/pasted. Defer to @djyou

Reply:
`no_push` appears twice because they are different in the two places. If a parameter is shared and has the same usage among multiple command groups, it's better to put it under root.

Reply:
Alternatively, you could define the metadata in a CLIArgumentType and simply go:
`c.argument('target', target_type)`

That way you can (if you want) be very specific with the scope but still have the same maintainability of only needing to change it once.